src/main/java/com/sprint/mission/discodeit/service/basic/BasicBinaryContentService.java

```diff
@@ -0,0 +1,43 @@
+package com.sprint.mission.discodeit.service.basic;
+
+import com.sprint.mission.discodeit.dto.binaryContent.BinaryContentCreateRequest;
+import com.sprint.mission.discodeit.entity.BinaryContent;
+import com.sprint.mission.discodeit.repository.BinaryContentRepository;
+import com.sprint.mission.discodeit.service.BinaryContentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class BasicBinaryContentService implements BinaryContentService {
+    private final BinaryContentRepository binaryContentRepository;
+
+    @Override
+    public BinaryContent create(BinaryContentCreateRequest request) {
+        BinaryContent content = new BinaryContent(
+                request.fileNm(),
```

Comment:
약어는 나중에 이해하기 힘들어지기에 지양하는게 좋을 것 같아요.  
```suggestion
                request.fileName(),
```